packages/utilities/exposure-scanning/args/index.js

```diff
@@ -0,0 +1,45 @@
+const isValidId = id => {
```

Comment:
Sorry probably should have suggested this library before, good one to avoid hand writing command line args parsing: https://www.npmjs.com/package/command-line-args

Reply:
Cool, thanks, will have a look